src/packages/admin-ui-components/src/filter-bar/component.tsx

```diff
@@ -51,7 +51,11 @@ export const FilterBar = ({ iconBefore }: { iconBefore?: ReactNode }) => {
 	const getFilterComponents = (entityName: string) => {
 		const rowEntity = entityByName(entityName);
 
-		return rowEntity.fields.map((field) => {
+		// @todo - currently the filters are not fitting on the screen
+		// we plan to redo this filter bar so that it is a drop down
+		// for now the workaround is to reduce the number of filters to 5
```

Comment:
Just need to communicate this to Neil/whoever's demoing so that we don't look afool should someone want >5 filters.

Reply:
Yeah, ran this past @thekevinbrown so he is aware.